python/TestHarness/testers/Tester.py

```diff
@@ -41,6 +41,8 @@ def validParams():
     params.addParam('petsc_debug',   ['ALL'], "{False,True} -> test only runs when PETSc is configured with --with-debugging={0,1}, otherwise test always runs.")
     params.addParam('curl',          ['ALL'], "A test that runs only if CURL is detected ('ALL', 'TRUE', 'FALSE')")
     params.addParam('tbb',           ['ALL'], "A test that runs only if TBB is available ('ALL', 'TRUE', 'FALSE')")
+    params.addParam('pthreads',      ['ALL'], "A test that runs only if PTHREADS is available ('ALL', 'TRUE', 'FALSE')")
+    params.addParam('openmp',        ['ALL'], "A test that runs only if OPENMP is available ('ALL', 'TRUE', 'FALSE')")
```

Comment:
We shouldn't need to check for these in the test harness.  The only reason to check is if we have a feature that explicitly only works with one model (like the NiCE integration stuff relying on TBB).  Other than that, we ALWAYS have at least one threading model enabled... but it's abstracted out.

Can you guys explain why we would need to ever check for OpenMP or Pthreads in the test harness?